test/e2e/cases/utils.go

```diff
@@ -273,8 +275,8 @@ func initDeps() (project, kubeConfigPath, lbName, ipName string, clients map[str
 	glog.Infof("Creating an mci named '%s' with ip address named '%s'", lbName, ipName)
 
 	// Reserve the IP address.
-	if _, err := kubeutils.ExecuteCommand([]string{"gcloud", "compute", "addresses", "create", "--global", ipName}); err != nil {
-		glog.Fatalf("Error creating IP address: %v", err)
+	if output, err := kubeutils.ExecuteCommand([]string{"gcloud", "compute", "addresses", "create", "--global", ipName}); err != nil {
+		glog.Fatalf("Error creating IP address: %v (output: %s)", err, output)
```

Comment:
err from ExecuteCommand should already have the output?

Reply:
Ah, ExecuteCommand does include 'output' in the error. BUT, the actual error from gcloud is not printed. I think we need to look at STDERR. (which is a problem I will leave for later.)